### PR TITLE
Pancreaticoduodenectomy knowledge

### PR DESCRIPTION
**Pull request by savva-k**
If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Describe the contribution to the taxonomy**

- ...
- ...
- ...

**Input given at the prompt**

```
   ...
```

**Response from the original model**

```
  ...
```

**Response from the fine-tuned model**

```
  ...
```

**Contribution checklist**

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [ ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [ ] The `qna.yaml` file contains at least 5 `seed_examples`
- [ ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [ ] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [ ] Content does not include PII or otherwise sensitive or confidential information
- [ ] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

**Comment by savva-k** [comment]
My branch name is incorrect, but I think it's not a big deal 🙂 I first wanted to cover wisdom teeth, but then found out it was taken by someone else.

**Comment by jjasghar** [comment]
Please fix the linting issues.

File: knowledge/science/medical/pancreaticoduodenectomy/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+      pancreas, duodenum and common bile duct necessitates en bloc resection of
+      these multiple structures. Other
+
+      indications for pancreaticoduodenectomy include chronic pancreatitis,
+      benign tumors of the pancreas, cancer
+
+      metastatic to the pancreas, multiple endocrine neoplasia type and
+      gastrointestinal stromal tumors
+  - question: Are there any other indications for a whipple procedure?
```
Comment (jjasghar) [inline knowledge/science/medical/pancreaticoduodenectomy/qna.yaml:37]:
```suggestion
  - question: What are other indications for a Whipple procedure?
```

File: knowledge/science/medical/pancreaticoduodenectomy/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
+created_by: savva-k
+domain: medical
+seed_examples:
+  - question: What is a pancreaticoduodenectomy?
+    answer: >
+      Pancreaticoduodenectomy is a major surgical operation most often performed
+      to remove cancerous tumours
+
```
Comment (jjasghar) [inline knowledge/science/medical/pancreaticoduodenectomy/qna.yaml:8]: Please remove these extra blank lines in this file.

File: knowledge/science/medical/pancreaticoduodenectomy/qna.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  - question: Can a head pancreas tumor be gently removed?
+    answer: >
+      Due to the shared blood supply of organs in the proximal gastrointestinal
+      system, surgical removal of the
+
+      head of the pancreas also necessitates removal of the duodenum, proximal
+      jejunum, gallbladder, and, occasionally,
+
+      part of the stomach
+  - question: What are main indications for a whipple procedure
```
Comment (jjasghar) [inline knowledge/science/medical/pancreaticoduodenectomy/qna.yaml:21]:
```suggestion
  - question: What are the main indications for a Whipple procedure?
```

File: knowledge/science/medical/pancreaticoduodenectomy/qna.yaml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+      metastatic to the pancreas, multiple endocrine neoplasia type and
+      gastrointestinal stromal tumors
+  - question: Are there any other indications for a whipple procedure?
+    answer: >
+      Pancreaticoduodenectomy is also used for the treatment of pancreatic or
+      duodenal trauma, or chronic
+
+      pancreatitis
+  - question: What are contradictions for a pancreaticoduodenectomy
```
Comment (jjasghar) [inline knowledge/science/medical/pancreaticoduodenectomy/qna.yaml:43]:
```suggestion
  - question: What are the contradictions of a pancreaticoduodenectomy?
```

**Review by jjasghar — CHANGES_REQUESTED** [review]
A few comments on the wording of the questions.

File: knowledge/science/medical/pancreaticoduodenectomy/qna.yaml (hunk 2 above)
Comment (savva-k) [reply knowledge/science/medical/pancreaticoduodenectomy/qna.yaml:8]: Fixed

File: knowledge/science/medical/pancreaticoduodenectomy/qna.yaml (hunk 4 above)
Comment (savva-k) [reply knowledge/science/medical/pancreaticoduodenectomy/qna.yaml]: Thank you for suggesting the edits

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please resolved the linting issues listed here: https://github.com/instructlab/taxonomy/pull/974/files

**Comment by jjasghar** [comment]
@instructlab-bot precheck

**Pull request closed without merging**